extensions/assetLibrary/package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "asset-library",
+  "version": "1.0.0",
+  "description": "Manage external dialog assets.",
+  "main": "lib/index.js",
+  "scripts": {
+    "build": "yarn clean && yarn build:client && yarn build:node",
+    "build:client": "webpack --config webpack.config.js",
+    "build:node": "tsc --project tsconfig.node.json",
```

Comment:
This needs to all go through webpack now.

Reply:
whoops somehow I didn't commit those changes from a few days ago.